OBSDOCS-675: Clean up logging installation CLI docs (#67450)
openshift/openshift-docs: modules/cluster-logging-deploy-console.adoc

:_mod-docs-content-type: PROCEDURE
[id="cluster-logging-deploy-console_{context}"]
= Installing the {clo} by using the web console

You can install the {clo} by using the {product-title} web console.

.Prerequisites

* You have administrator permissions.
* You have access to the {product-title} web console.

.Procedure

. In the {product-title} web console, click *Operators* -> *OperatorHub*.
. Type `OpenShift Logging` in the *Filter by keyword* box.
. Choose  *Red Hat OpenShift Logging* from the list of available Operators, and click *Install*.
. Ensure that *A specific namespace on the cluster* is selected under *Installation mode*.

> . You must select *openshift-logging* as the *Installed Namespace* for logging versions 5.7 and older. For logging 5.8 and later versions, you can use any namespace.

Reviewer: There isn't no different as before. Please revert line 22.

. Select *Enable operator recommended cluster monitoring on this namespace*.
+

> This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-logging` namespace.

Reviewer 2: "Scrapes" seems a bit out of place when it comes to translation. Would "scans" be more suitable?

Author: It's generally referred to as "scraping" metrics 🙂 see https://prometheus.io/docs/prometheus/latest/getting_started/

.Verification

. Verify that the *Red Hat OpenShift Logging Operator* is installed by switching to the *Operators* → *Installed Operators* page.

> . In the *Status* column, verify that you see green checkmarks with *InstallSucceeded* and the text *Up to date*.

Reviewer 2: Are *InstallSucceeded* and *Up to date* fields? Maybe the word "field" could be implemented so that both these entries do not seem slightly different.

Author: Not sure, since this came from preexisting content

[IMPORTANT]
====
An Operator might display a `Failed` status before the installation finishes. If the Operator install completes with an `InstallSucceeded` message, refresh the page.
====

> If the Operator does not appear as installed, to troubleshoot further:

Reviewer 2: ```suggestion
If the Operator does not show as installed, to troubleshoot further:
```

![Screenshot from 2023-12-18 15-15-04](https://github.com/openshift/openshift-docs/assets/57954076/d87c0ca0-33a3-497c-8daf-fb804c15fa98)

Reviewer 2: ```suggestion
If the Operator does not appear as installed, choose one of the following troublshooting options:
```

> * Switch to the *Operators* → *Installed Operators* page and inspect the *Status* column for any errors or failures.

Reviewer 2: ```suggestion
* Switch to the *Operators* → *Installed Operators* page, and inspect the *Status* column for any errors or failures.
```
